src/Symfony/Component/Uuid/Tests/UuidTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uuid;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Uuid\Uuid;
+
+/**
+ * @requires function uuid_is_valid
```

Comment:
can be removed since the polyfill does it, isn't it?

Reply:
If you git clone symfony/symfony (not the component) and you don't have the UUID ext install (nor the polyfill) it will fails

Reply:
the polyfill should be added to the root metapackage instead.